test/visibility/private/moduleSymbols/accessPrivateUsedSibling2.chpl

```diff
@@ -0,0 +1,23 @@
+module parent {
+  private module child {
+    proc secretFunction(a: int) {
+      return a*3;
+    }
+
+  }
+
+  // 'use's are private by default, at least to external modules.  So
+  // should 'sibling' be able to reference child's contents when it
+  // 'use's 'parent' without its own use or import of child's
+  // contents?  Interestingly, variables and functions seem to be
+  // treated differently.
```

Comment:
This comment seems out of date

Reply:
Good point, thanks.  (it's good to be reminded why I don't like writing comments :) ).